GiniVision/Classes/ReviewContainerViewController.swift

```diff
@@ -51,26 +47,20 @@ internal class ReviewContainerViewController: UIViewController, ContainerViewCon
         view.backgroundColor = GiniConfiguration.sharedConfiguration.backgroundColor
         
         // Configure back button
-        backButton = GiniBarButtonItem(
-            image: backButtonImage,
-            title: GiniConfiguration.sharedConfiguration.navigationBarReviewTitleBackButton,
```

Comment:
It's a good point. I'll look into keeping the API. We shouldn't break old implementations.